scripts/install-helpers/baremetal-coco/install.sh

```diff
@@ -0,0 +1,733 @@
+kind: KataConfig
+metadata:
+  name: cluster-kataconfig
+spec:
+  enablePeerPods: false
+  logLevel: info
+  kataConfigPoolSelector:
+    matchLabels:
+      $label
+EOF
+
+    result=$?
+    [ $result -eq 0 ] || return 1
+
+    echo "KataConfig object successfully created"
+    return 0
+}
+
+# Function to set aa_kbc_params for teh Kata agent to be used for
+# attestation
+# Use drop-in configuration via MachineConfig
+# This function accepts the TEE type as an argument
+# It also accepts the Trustee URL as an argument
+# Trustee URL should be complete URL with the form http(s)://<trustee_ip>:<port>
+function set_aa_kbc_params_for_kata_agent() {
+    local tee_type=${1}
+    local trustee_url=${2}
+    local source=""
+    local filepath=""
+
+    # Create kata configuration toml override for the kernel_params
+    # Input kernel_params="agent.aa_kbc_params=cc_kbc::$trustee_url"
+    kata_override="[hypervisor.qemu]
+kernel_params= \"agent.aa_kbc_params=cc_kbc::$trustee_url\""
+
+    # Create base64 encoding of the drop-in to be used as source
+    source=$(echo "$kata_override" | base64 -w0) || return 1
+
+    # This is applied after Kata installation so we should use the kata-oc label
+    # for worker nodes
+    local mc_label="machineconfiguration.openshift.io/role: kata-oc"
+
+    if is_single_node_ocp; then
+        mc_label="machineconfiguration.openshift.io/role: master"
+    fi
+
+    case $tee_type in
+    tdx)
+        filepath=/etc/kata-containers/tdx/config.d/96-kata-kernel-config
+        ;;
+    snp)
+        filepath=/etc/kata-containers/snp/config.d/96-kata-kernel-config
+        ;;
+    esac
+
+    cat <<EOF >./96-kata-kernel-config-mc.yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    $mc_label
+  name: 96-kata-kernel-config
+  namespace: openshift-machine-config-operator
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,$source
+        mode: 420
+        overwrite: true
+        path: $filepath
+EOF
+
+    oc apply -f 96-kata-kernel-config-mc.yaml || return 1
+
+}
+
+# Function to create runtimeClass based on TEE type and
+# SNO or regular OCP
+# Generic template
+#apiVersion: node.k8s.io/v1
+#handler: kata-$TEE_TYPE
+#kind: RuntimeClass
+#metadata:
+#  name: kata-$TEE_TYPE
+#scheduling:
+#  nodeSelector:
+#    $label
+function create_runtimeclasses() {
+    local tee_type=${1}
+    local label='node-role.kubernetes.io/kata-oc: ""'
+    local ext_resources=''
+
+    if is_single_node_ocp; then
+        label='node-role.kubernetes.io/master: ""'
+    fi
+
+    case $tee_type in
+    tdx)
+        ext_resources='tdx.intel.com/keys: 1'
+        ;;
+    snp)
+        ext_resources='sev-snp.amd.com/esids: 1'
+        ;;
+    esac
+
+    echo "Creating kata-$tee_type RuntimeClass object"
+
+    #Create runtimeClass object
+    oc apply -f - <<EOF
```

Comment:
If you do :
```suggestion
    oc apply -f - || return 1 <<EOF
```
you can...